File: api/ActivitySignUpModels/Activity/ActivitySignedUpViewModel.cs
using ActivitySignUp.Models.Comment;
using ActivitySignUp.Models.Person;
using System;
using System.Collections.Generic;


namespace ActivitySignUp.Models.Activity
{
    public class ActivitySignedUpViewModel
    {
        public int ActivityId { get; set; }
        public string ActivityName { get; set; }
        public string ActivityDescription { get; set; }
        public DateTime ActivityDateTime { get; set; }
        public string ActivityImage { get; set; }
        public List<PersonListModel> ParticipantList { get; set; }

        public List<CommentListModel> CommentList { get; set; }
    }
}

File: client/ActivitySignUpPortal/src/app/models/activity/ActivitySignedUpViewModel.ts
import { Deserializable } from '../deserializable.model';
import { PersonListModel } from "../person/PersonListModel";
import { CommentListModel } from "../comment/CommentListModel";

export class ActivitySignedUpViewModel implements Deserializable {
     activityId: number | undefined;
     activityName: string | undefined;
     activityDescription: string | undefined;
     activityDateTime: Date | undefined;
     activityImage: File | undefined;
     participantList: Array<PersonListModel> | undefined;
     commentList: Array<CommentListModel> | undefined;

    constructor(
 
    ){}

    deserialize(input: any): this {
        Object.assign(this, input);
         return this;
    }


}